### ISSUE TITLE
Can you provide the uncompiled raw markdown file for the beginners

### ISSUE THREAD

**Issue by chthub** [panglesd/slipshow#55]
Very nice work, but it's still hard for beginners to create presentation, can you provide the examples with uncompiled raw markdown file (not html file)? so we could learn and make modifications. Thanks!

**Comment by panglesd** [comment]
Thanks a lot!

Yes, it seems this repository lacks example a lot! I'll try to add some tomorrow, I'll update on on this issue as soon as I have something.

(I've made many presentations with slipshow, but most of them were written directly in html, before the markdown compiler existed. That's why I don't have a lot of markdown content to show. I do have some, that I have to clean up a little bit before they can be featured as examples!)

**Comment by panglesd** [comment]
Hello!

I've pushed some examples of presentations I made, written in markdown, in the [example folder](https://github.com/panglesd/slipshow/tree/master/example/ocaml_lecture).
They are obviously not perfect, but they can serve as examples to understand the syntax.

**Comment by chthub** [comment]
> Hello!
> 
> I've pushed some examples of presentations I made, written in markdown, in the [example folder](https://github.com/panglesd/slipshow/tree/master/example/ocaml_lecture). They are obviously not perfect, but they can serve as examples to understand the syntax.

Good example, I still have two questions:
1. how to implement the effects that one large page includes many subpages, like this one:
![image](https://github.com/panglesd/slipshow/assets/45276259/054afac6-26e3-4abe-a6ff-63325fcd30b2)

2. Another question, how to implement the presentation in this page (by zooming to see previous pages):
![image](https://github.com/panglesd/slipshow/assets/45276259/8935a51c-70fa-4c93-99ce-a4c4c061d636)

**Comment by panglesd** [comment]
> how to implement the effects that one large page includes many subpages, like this one:

This is #28 and is unfortunately not yet done. Wat you can do is to use embedded html, like so:
```
markdown content

<slip-slip auto-enter scale=0.4 delay=1>
  <slip-body>

markdown content for the inside slip

  </slip-body>
</slip-slip>
```

This is clearly not ideal, and I hope to solve that soon! In the meantime, missing features can always be implemented with embedded html, and having a look at the handwritten html should provide the examples.

> Another question, how to implement the presentation in this page (by zooming to see previous pages):

This was made with a much earlier version of the engine (using a deprecated way of triggering actions).
The code responsible for this is:

<details>
<summary> Old code for unzooming </summary>

```
	 future.setNthAction(10, (slip) => {
	     if(typeof slip.savedX == "undefined") {
		 slip.savedX = slip.currentX; // saving initial coordinates inside slip
		 slip.savedY = slip.currentY; // so that they are accessible in the next action
	     }
	     setTimeout(()=> {
		 slip.moveWindow(2,2,4,0,1);	 
	     }, 0);
	 });
```

</details>

Now, there is the `zoom-at` and `unzoom-at` attributes, but I don't think they can work outside of the current slip. If you need this feature, I can find a hack which will work!

---

It is clear that I should rewrite all examples in markdown and with the latest engine, so that the examples are actual example people can use...

**Comment by chthub** [comment]
> > how to implement the effects that one large page includes many subpages, like this one:
> 
> This is #28 and is unfortunately not yet done. Wat you can do is to use embedded html, like so:
> 
> ```
> markdown content
> 
> <slip-slip auto-enter scale=0.4 delay=1>
>   <slip-body>
> 
> markdown content for the inside slip
> 
>   </slip-body>
> </slip-slip>
> ```
> 
> This is clearly not ideal, and I hope to solve that soon! In the meantime, missing features can always be implemented with embedded html, and having a look at the handwritten html should provide the examples.
> 
> > Another question, how to implement the presentation in this page (by zooming to see previous pages):
> 
> This was made with a much earlier version of the engine (using a deprecated way of triggering actions). The code responsible for this is:
> 
> Old code for unzooming
> Now, there is the `zoom-at` and `unzoom-at` attributes, but I don't think they can work outside of the current slip. If you need this feature, I can find a hack which will work!
> 
> It is clear that I should rewrite all examples in markdown and with the latest engine, so that the examples are actual example people can use...

Thank you for your reply! I have made two presentations using Slipshow, and it works well. I like it. Currently, I have encountered a few issues:
1. The VS Code extension of Slipshow does not display images in preview mode.
2. When I share the single HTML file with my friends, the images are missing. It's impractical to share an additional image folder. I'm not sure how to convert the format to include all contents for easier sharing.

**Comment by panglesd** [comment]
Indeed... I opened https://github.com/panglesd/slipshow-vscode/issues/1 to track that.

In the meantine, if you have a Linux or Mac, you can use the precompiled binaries, where the images should work both in preview and compile mode.

**Comment by panglesd** [comment]
I just wanted to mention that with the newly released version of the slipshow extension for VSCode (0.0.3), the problem with images not appearing in the preview, and not being fully included in the HTML file, should be finally fixed.

**Comment by chthub** [comment]
> I just wanted to mention that with the newly released version of the slipshow extension for VSCode (0.0.3), the problem with images not appearing in the preview, and not being fully included in the HTML file, should be finally fixed.

Good to know, thanks for doing that!
